etc/config.sample.toml

```diff
@@ -124,6 +124,11 @@
   # will allow TSM compactions to write to disk.
   # compact-throughput-burst = "48m"
 
+  # CompactPointsPerBlock is the points per block to be used when aggressive
```

Comment:
I think this should be `AggressivePointsPerBlock` or something like that so we know it is not what is used normally.